File: 프로그래머스/Lv.0/진료순서 정하기/120835.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] emergency) {
        int[] answer = new int[emergency.length];
        int[] arr = Arrays.copyOf(emergency, emergency.length);
        
        Arrays.sort(arr);
        
        for (int i=0; i<emergency.length; i++) {
            for (int j=0; j<arr.length; j++) {
                if (emergency[i] == arr[j]) {
                    answer[i] = emergency.length - j;
                }
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/Lv.0/진료순서 정하기/120835.js
function solution(emergency) {
    const answer = [];
    const arr = [...emergency].sort((a, b) => b - a);
    
    emergency.forEach((v, i) => {
        for (let j=0; j<arr.length; j++) {
            if (v === arr[j]) {
                answer[i] = j + 1;
                break;
            }
        }
    });
    
    return answer;
}
